src/transactions/InvokeHostFunctionOpFrame.cpp

```diff
@@ -603,6 +603,12 @@ InvokeHostFunctionOpFrame::doApply(Application& app, AbstractLedgerTxn& ltx,
         innerResult().code(INVOKE_HOST_FUNCTION_RESOURCE_LIMIT_EXCEEDED);
         return false;
     }
+    metrics.mEmitEventByte += out.result_value.data.size();
+    if (resources.contractEventsSizeBytes < metrics.mEmitEventByte)
```

Comment:
I thought `contractEventsSizeBytes` is being removed due to your xdr change?

Reply:
This PR was created prior to the XDR change, but XDR change only removes the value from `resources`. So condition here will need to change to `config.txMaxContractEventsSizeBytes`.

Reply:
Actually, I'll probably integrate the XDR change in this PR to keep things simpler (and also there is something suspicious going on with the fees, I need to investigate that as well). I'll ping this when ready.

Reply:
The PR is updated and is ready for review again.